apps/omg/test/support/test_helper.ex

```diff
@@ -99,6 +99,17 @@ defmodule OMG.TestHelper do
     create_signed(inputs, outputs) |> Transaction.Signed.encode()
   end
 
+  def create_encoded_fee_tx(blknum, owner, currency, amount) do
```

Comment:
this is used only in `happy_path_test.exs `, why is this in a test_helper?

Reply:
Good question :thinking: